test/nx_signal/filters_test.exs

```diff
@@ -1,4 +1,46 @@
 defmodule NxSignal.FiltersTest do
   use NxSignal.Case
   doctest NxSignal.Filters
+
+  describe "median/2" do
```

Comment:
You should also add happy-path tests

Reply:
Definitely, I'm getting small discrepancies with scipy results so I wanted to hold on that

Reply:
I wouldn't worry about small discrepancies. If the result is theoretically correct aside from rounding errors, you can use it